src/main/webapp/app/exercises/shared/feedback/feedback.component.html

```diff
@@ -44,7 +44,7 @@ <h4>
                         </ng-template>
                     </h4>
 
-                    <span class="badge" [ngClass]="badge.class" id="result-score-badge" ngbTooltip="{{ badge.tooltip | artemisTranslate }}">
+                    <span *ngIf="badge" class="badge" [ngClass]="badge.class" id="result-score-badge" ngbTooltip="{{ badge.tooltip | artemisTranslate }}">
```

Comment:
this was a issue, that actually does not have anything to do directly with this PR but that I noticed and quick-fixed.

I will open a follow-up PR that deals with these status badges in general and investigate this further